src/shared/lib/StoreUtils.ts

```diff
@@ -85,6 +99,31 @@ export async function fetchGenes(hugoGeneSymbols?: string[],
     }
 }
 
+export async function fetchReferenceGenomeGenes(genomeName:string, hugoGeneSymbols?: string[],
+                                 client: CBioPortalAPI = defaultClient)
+{
+    if (hugoGeneSymbols && hugoGeneSymbols.length) {
+        const order = stringListToIndexSet(hugoGeneSymbols);
+        return _.sortBy(await client.fetchGenesUsingPOST_1({
```

Comment:
@onursumer noticed this but it seem slike fetchGenesUsingPost is overloaded (on backend) which produces these indexed methods on client.   We need to better distinguish referenceGenomeGenes from genes.

Reply:
@alisman @khzhu I have added the relevant review for this one here: https://github.com/cBioPortal/cbioportal/pull/5891

Reply:
thanks, @onursumer! Those notes I put on schema diagram might help with the review.